src/tox/session/__init__.py

```diff
@@ -220,6 +220,26 @@ def subcommand_test(self):
             retcode = self._summary()
         return retcode
 
+    def _add_parallel_summaries(self):
+        import json
+        if 'testenvs' not in self.resultlog.dict:
+            return
+        for venv in self.venv_dict.values():
+            testenvs = self.resultlog.dict['testenvs']
+            if venv.name not in testenvs:
+                continue
+            result_json_path = venv.path.join('.tox-result.json')
+            if result_json_path.exists():
+                with result_json_path.open("rb") as f:
+                    data = json.loads(f.read())
+                if not data:
+                    continue
+                if 'testenvs' not in data:
+                    continue
+                if venv.name not in data['testenvs']:
+                    continue
+                testenvs[venv.name] = data['testenvs'][venv.name]
```

Comment:
on merge we should delete the child result json

Reply:
I was thinking about that, but other parts of tox like the log output don't do that, so I wasn't sure. I don't see any harm in keeping it. I think it makes sense to delete it before starting the sub-process though.

Reply:
I would prefer to delete it, to make sequential and parallel run behave the same, aka in the end produce a single result log file 🤔